src/app/shared/components/NFTBoosterCard/SelfContainedNFTBoosterCard.tsx

```diff
@@ -2,16 +2,22 @@ import { useNFTBoosterContext } from '@/app/providers/NFT/BoosterContext'
 import { NFTBoosterCard } from '@/app/shared/components'
 import { FC } from 'react'
 import { communitiesMapByContract } from '../../../communities/communityUtils'
+import { Address, isAddressEqual } from 'viem'
 
 export const SelfContainedNFTBoosterCard: FC = () => {
   const { hasActiveCampaign, currentBoost, boostData } = useNFTBoosterContext()
-  const { title } = communitiesMapByContract[boostData?.nftContractAddress ?? ''] ?? {}
+  const [, community] =
+    Object.entries(communitiesMapByContract).find(
```

Comment:
With this one, we may change the component from the dao team, but I was not sure about the side effects, so I decided to find it again.